compiler/lcalc/optimizations.ml

```diff
@@ -74,9 +137,10 @@ let peephole_optimizations (p : program) : program =
         (fun scope_body ->
           {
             scope_body with
-            scope_body_expr = Bindlib.unbox (peephole_expr scope_body.scope_body_expr);
+            scope_body_expr = Bindlib.unbox (peephole_expr () scope_body.scope_body_expr);
           })
         p.scopes;
   }
 
-let optimize_program (p : program) : program = peephole_optimizations p
+let optimize_program (p : program) : program = p |> iota_optimizations |> peephole_optimizations
+(* |> beta_optimizations *)
```

Comment:
Can you explain why you disable beta reduction in a few words in a comment? Like we don't want to inline everything, etc.